packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerHeader.tsx

```diff
@@ -12,13 +12,15 @@ const StyledContainer = styled.div`
   display: flex;
   height: ${({ theme }) => theme.spacing(8)};
   user-select: none;
+  width: calc(100% - ${({ theme }) => theme.spacing(1.5)});
```

Comment:
nipick: I think we try to avoid .5 in spacing to only have multiples of 4px except for rare exceptions where we can't do otherweise. I believe the the width is more `width: calc(100% - ${({ theme }) => theme.spacing(2)});` and then we wouldn't need a half-sized padding? Approving so that you can merge independently after you've made the call

Reply:
agree!

Reply:
good !